Card/index.spec.ts

```diff
@@ -120,6 +120,10 @@ describe("@pax2pay/model.Card", () => {
 		expect(model.Card.Token.is(token)).toBeTruthy()
 		token = "4567897890/16/0221/220115/hgUVP-7IwsW963dqwG0sop71sE4F5rCMJ85DJYnwWeGVYdXp/EfyBTD89WFZOfGGq4UGg-w"
 		expect(model.Card.Token.is(token)).toBeTruthy()
```

Comment:
This is a long term token that already worked. Why what is different with the other once?

Reply:
encrypted length is different
current short term's encrypted length is 48 & long term's is 40
but in unit test, they are all 48, 
that's why we failed when validate long term token